Sources/App/Views/Keyword/KeywordShow+View.swift

```diff
@@ -66,8 +79,16 @@ enum KeywordShow {
                 )
             )
         }
-    }
 
+        private func showPackageLimitDisclaimer() -> Node<HTML.BodyContext> {
+            guard model.totalPackageCount >= Constants.maxKeywordPackageCollectionCount else { return .empty }
+            return .p(
+                .h6(
```

Comment:
Let's not put this in an `h6`, it can just be a `p` tag.

I'd suggest this text:

```html
<p><strong>Note:</strong> Package collections are limited to a maximum of 300 items and this keyword has more than 300 packages associated with it. This collection includes only the top 300 packages for this keyword sorted by package score.</p>
```